03 State/src/app.jsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import {HelloComponent} from './hello';
+import {NameEditComponent} from './nameEdit';
+
+interface Props {
+}
+
+interface State {
+  userName : string;
+}
+
+export class App extends React.Component<Props, State> {
```

Comment:
Does this extends <> work fine in Es6?

https://toddmotto.com/react-create-class-versus-component/